core/src/main/java/org/bitcoinj/base/internal/ByteUtils.java

```diff
@@ -203,31 +203,6 @@ public static void writeUint32BE(long val, byte[] out, int offset) throws ArrayI
         writeUint32BE(val, ByteBuffer.wrap(out, offset, out.length - offset));
     }
 
-    /**
```

Comment:
Do we want to remove these methods if we don't use them? They might be useful for later usecases?